### ISSUE TITLE
Administrador não consegue ver as campanhas disponíveis

### ISSUE THREAD

**Issue by frchico** [CBSIIFSLagarto/VotoIFS#34]
Permitir que o administrador visualize as campanhas que estão disponíveis

**Comment by willfarias141** [comment]
@frchico Não entendi. É para o adm ver todas as campanhas cadastradas no sistema, de todos professores? PS.: ficou definido que o adm não cadastra campanha.

**Comment by frchico** [comment]
Verdade, não tinha lembrado deste requisito. Vai ficar para a próxima turma ;)
